packages/victory-core/src/victory-animation/victory-animation.tsx

```diff
@@ -163,21 +236,21 @@ export default class VictoryAnimation extends React.Component {
         setTimeout(() => {
           this.loopID = this.timer.subscribe(
             this.functionToBeRunEachFrame,
-            this.props.duration
+            this.props.duration!
```

Comment:
We have a general problem with our types.  Since we use `defaultProps`, some of our types are optional to the "outside world" but are always defined internally.

Typically, I solve this in FC's by using defaults when deconstructing.  Not sure if there's a better way to handle that for Class Components.